src/app/App.ts

```diff
@@ -16,5 +15,4 @@ export const App = (): void => {
   const chessModel: ChessModel = new ChessEngine(chessBoard);
   const presenter = new ChessBoardPresenter(chessBoardView, chessModel);
   presenter.startGame();
```

Comment:
@nowakprojects wgl trochę dla mnie jest niejasne, że on faktycznie otrzymuje obiekt z pozycjami pionków, a nie pusty... czegoś jest wg mnie za dużo... bo tutaj wyżej mam, że `const boardWithPieces: SquareWithPiece = {};` jest puste i to przekazuję do ChessBoard, który z kolei przekazuję do ChessEngine, w którym istenieje coś takiego jak boardWithPieces tylko pod inną nazwą. Dobrze mówię, że tutaj jest czegoś za dużo?